nodetypes/radon.nodes.aws/AwsLambdaFunctionFromS3/NodeType.tosca

```diff
@@ -29,6 +28,9 @@ node_types:
         type: integer
         description: Function memory in MB
         default: 128
+      function_name:
```

Comment:
@sydkbc there's no need to define the `function_name` property because the property `name` in abstract Function NT is sued for that reason.

Reply:
also, since the radon.nodes.aws.AwsLambdaFunctionFromS3Scalable NT was introduced, is there a reason why this old type needs to be changed?

Reply:
function_name was introduced as a new property for the monitoring functionality.
So if we need to remove it, updates have to be performed on the AWSismonitored relationship as well.

If the change is indeed needed I will need some time to adapt the particles and recheck the GMT genration and the xOpera SaaS deployments.

Any proposals?